portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,299 @@
+          'Lead team in initial phase of codebase modernization, using React.JS, and redesign database structure'
+        ]
+      }
+    ],
+    'Activities': [
+      {
+        organization: 'Society of Hispanic Professional Engineers USF Chapter',
+        start: 'May 2020',
+        title: 'Vice-president of External Affairs',
+        location: 'Tampa, FL, USA'
+      },
+      {
+        organization: 'FIRST Global',
+        start: 'June 2019',
+        title: 'Team Venezuela Mentor',
+        location: 'Remote',
+        description: [
+          'Lead team marketing and external communications with the FIRST Global organization, sponsors, and the community, through the handling of emails, creation and frequent publication content on social media, official team video editing, uniform design, schedule organization, record keeping, etc.'
+        ]
+      },
+      {
+        organization: 'Carrollwood Day School',
+        start: 'August 2019',
+        end: 'January 2020',
+        title: 'FTC Team Nova #11343 Mentor',
+        location: 'Tampa, FL, USA',
+        description: [
+          'Assist in the academic preparation of a team of 5-7 students, as well as the building and programming of robot for season competitions'
+        ]
+      },
+    ]
+  }
+}
+
+document.addEventListener('DOMContentLoaded', main);
+
+/**
+ * Function to be run after the HTML body has been laoded
+ */
+function main() {
+  loadContactData(RESUME.contactData);
+  loadSectionData(RESUME.sections)
+}
+
+/**
+ * Processes and appends the given contact information on the website header.
+ * 
+ * @param {!object} contactData Dictionary containing the contact data to be shown. 
+ *    Its keys contain the name of each contact section, and its values are
+ *    either a single or an array of contact data.
+ */
+function loadContactData(contactData) {
+  const contactsContainer = document.querySelector('header > .contact-container');
+  const entries = createContactEntries(contactData);
+
+  contactsContainer.append(...entries);
+}
+
+/**
+ * This function converts each section in the given contact data to 
+ * an html node based on the "contact-entry" template.
+ * 
+ * @param {!object} contactData Dictionary containing the contact data to be shown. 
+ *    Its keys contain the name of each contact section, and its values are
+ *    either a single or an array of contact data.
+ * 
+ * @returns {!Array} The array of "contact-entry" html nodes.
+ */
+function createContactEntries(contactData) {
+  const contactEntryTemplate = document.querySelector('#contact-entry');
+  const entries = Object.entries(contactData).map(
+    ([title, data]) => loadContactEntry(contactEntryTemplate.content.cloneNode(true), title, data)
+  );
+
+  return entries;
+}
+
+/**
+ * This function populates an empty "contact-entry" html node given the contact data. 
+ * 
+ * @param {!DocumentFragment} entry The empty contact-entry html node.
```

Comment:
I would call this an `entryFragment` / `fragment`

Reply:
Done!